TouchID.m

```diff
@@ -6,16 +6,19 @@ @implementation TouchID
 
 RCT_EXPORT_MODULE();
 
-RCT_EXPORT_METHOD(authenticate: (RCTResponseSenderBlock)callback)
+RCT_EXPORT_METHOD(authenticate: (NSString *)reason
+                  callback: (RCTResponseSenderBlock)callback)
 {
     LAContext *context = [[LAContext alloc] init];
     NSError *error;
-    
+  if ( [[reason stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceCharacterSet]] length] == 0 ){
```

Comment:
Can you remove the padding within your if conditional.  You don't pad conditionals anywhere else, and neither does the previous committed code.